examples/office/importers/acme/acme_pdf.py

```diff
@@ -45,18 +29,9 @@ def pdf_to_text(filename):
     Returns:
       A string, the text contents of the filename.
     """
-    try:
-        pipe = subprocess.Popen(['pdf2txt.py', filename],
-                                stdout=subprocess.PIPE,
-                                stderr=subprocess.PIPE)
-    except (FileNotFoundError, PermissionError):
-        pipe = subprocess.Popen(['pdf2txt', filename],
-                                stdout=subprocess.PIPE,
-                                stderr=subprocess.PIPE)
-    stdout, stderr = pipe.communicate()
-    if stderr:
-        raise ValueError(stderr.decode())
-    return stdout.decode()
+    r = subprocess.run(['pdftotext', filename, '-'],
```

Comment:
Is it clean what happens if pdtotext isn't installed on the user system?

Reply:
You get an exception, same if `pdf2txt` is not installed. I think it is very likely that `pdftotext` is present on a desktop installation, but I don't have data to support this. I don't think it is worth implementing conditional tests in this new mechanism.